.ci/test-aarch64.yml

```diff
@@ -42,7 +42,7 @@ services:
       - TRAVIS_REPO_SLUG
 
   redis:
-    image: redis:3.2
+    image: redis:${REDIS_VERSION:-5}
```

Comment:
Why do we default to redis 5 if we're only testing v3 and v4?

Reply:
Redis 5 is tested in `Integration tests` > `Node v8`. Because this is the new default version.